src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -7115,6 +7157,43 @@ private BoundExpression BindIndexedPropertyAccess(SyntaxNode syntax, BoundExpres
 
                 if (!analyzedArguments.HasErrors)
                 {
+                    // PROTOTYPE: REMOVE BEFORE SHIPPING DEV16
```

Comment:
This is checking into shipping branches so no more prototype comments. Please file a bug instead to track this.